asserter/block.go

```diff
@@ -293,12 +293,31 @@ func (a *Asserter) Operations(
 		return ErrNoOperationsForConstruction
 	}
 
+	paymentTotal := big.NewInt(0)
+	feeTotal := big.NewInt(0)
+	paymentCount := 0
+	feeCount := 0
+	zero := big.NewInt(0)
 	for i, op := range operations {
 		// Ensure operations are sorted
 		if err := a.Operation(op, int64(i), construction); err != nil {
 			return err
 		}
 
+		if a.validations.Enabled {
+			if op.Type == a.validations.Payment.Name {
+				val, _ := new(big.Int).SetString(op.Amount.Value, 10)
```

Comment:
I think the result boolean matters, which indicate conversion succeed or not.

Reply:
We are already verifying this with the `Operation` call at line 303. 
https://github.com/coinbase/rosetta-sdk-go/blob/master/asserter/block.go#L53 - so we don't need it here